qcms coverage report is broken

I recently got:
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=31871

which fails with:
<details>

```
Step #3: Already have image: gcr.io/oss-fuzz/qcms
Step #3: ---------------------------------------------------------------
Step #3: Compiling libFuzzer to /usr/lib/libFuzzingEngine.a...  done.
Step #3: ---------------------------------------------------------------
Step #3: CC=clang
Step #3: CXX=clang++
Step #3: CFLAGS=-O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION  -fprofile-instr-generate -fcoverage-mapping -pthread -Wl,--no-as-needed -Wl,-ldl -Wl,-lm -Wno-unused-command-line-argument
Step #3: CXXFLAGS=-O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION  -fprofile-instr-generate -fcoverage-mapping -pthread -Wl,--no-as-needed -Wl,-ldl -Wl,-lm -Wno-unused-command-line-argument -stdlib=libc++
Step #3: ---------------------------------------------------------------
Step #3: + cargo fuzz build -O
Step #3:  Downloading crates ...
Step #3:   Downloaded libc v0.2.77
Step #3:   Downloaded cc v1.0.60
Step #3:   Downloaded arbitrary v0.4.6
Step #3:   Downloaded libfuzzer-sys v0.3.4
Step #3:    Compiling libc v0.2.77
Step #3:    Compiling cc v1.0.60
Step #3:    Compiling qcms v0.2.0 (/src/mozilla-central/gfx/qcms)
Step #3:    Compiling arbitrary v0.4.6
Step #3:    Compiling libfuzzer-sys v0.3.4
Step #3:    Compiling qcms-fuzz v0.0.0 (/src/mozilla-central/gfx/qcms/fuzz)
Step #3:     Finished dev [unoptimized + debuginfo] target(s) in 5.64s
Step #3: + find fuzz -iname '*.icc' -type f -exec zip -qju /workspace/out/coverage/fuzz_seed_corpus.zip '{}' ';'
Step #3: + cp fuzz/qcms_fuzzer.dict /workspace/out/coverage/fuzz.dict
Step #3: + cp fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_target_qcms /workspace/out/coverage/fuzz
Finished Step #3
Starting Step #4
Step #4: Pulling image: gcr.io/oss-fuzz-base/base-runner
Step #4: Using default tag: latest
Step #4: latest: Pulling from oss-fuzz-base/base-runner
Step #4: 4007a89234b4: Already exists
Step #4: 5dfa26c6b9c9: Already exists
Step #4: 0ba7bf18aa40: Already exists
Step #4: 4c6ec688ebe3: Already exists
Step #4: 4b6ec4143739: Already exists
Step #4: cda5d15d1ded: Already exists
Step #4: ae2d7a5fbd27: Pulling fs layer
Step #4: 0fa637aa9847: Pulling fs layer
Step #4: 2364234e9070: Pulling fs layer
Step #4: d0e039795b49: Pulling fs layer
Step #4: acfaef75e502: Pulling fs layer
Step #4: d0e039795b49: Waiting
Step #4: b0bddfe7c840: Pulling fs layer
Step #4: acfaef75e502: Waiting
Step #4: 6ec7897618e7: Pulling fs layer
Step #4: 5587c4496d81: Pulling fs layer
Step #4: 6ec7897618e7: Waiting
Step #4: 9bb812f79feb: Pulling fs layer
Step #4: 5587c4496d81: Waiting
Step #4: 15afa30ad4cd: Pulling fs layer
Step #4: 9bb812f79feb: Waiting
Step #4: 15afa30ad4cd: Waiting
Step #4: ae2d7a5fbd27: Verifying Checksum
Step #4: ae2d7a5fbd27: Download complete
Step #4: 0fa637aa9847: Verifying Checksum
Step #4: 0fa637aa9847: Download complete
Step #4: d0e039795b49: Download complete
Step #4: ae2d7a5fbd27: Pull complete
Step #4: acfaef75e502: Verifying Checksum
Step #4: acfaef75e502: Download complete
Step #4: 6ec7897618e7: Verifying Checksum
Step #4: 6ec7897618e7: Download complete
Step #4: 0fa637aa9847: Pull complete
Step #4: 5587c4496d81: Verifying Checksum
Step #4: 5587c4496d81: Download complete
Step #4: b0bddfe7c840: Verifying Checksum
Step #4: b0bddfe7c840: Download complete
Step #4: 15afa30ad4cd: Verifying Checksum
Step #4: 15afa30ad4cd: Download complete
Step #4: 2364234e9070: Verifying Checksum
Step #4: 2364234e9070: Download complete
Step #4: 9bb812f79feb: Verifying Checksum
Step #4: 9bb812f79feb: Download complete
Step #4: 2364234e9070: Pull complete
Step #4: d0e039795b49: Pull complete
Step #4: acfaef75e502: Pull complete
Step #4: b0bddfe7c840: Pull complete
Step #4: 6ec7897618e7: Pull complete
Step #4: 5587c4496d81: Pull complete
Step #4: 9bb812f79feb: Pull complete
Step #4: 15afa30ad4cd: Pull complete
Step #4: Digest: sha256:c67cc329110a39426afb68628c3b091473310857117ba401826fb1d493e41b17
Step #4: Status: Downloaded newer image for gcr.io/oss-fuzz-base/base-runner:latest
Step #4: gcr.io/oss-fuzz-base/base-runner:latest
Finished Step #4
Starting Step #5
Step #5: Already have image (with digest): gcr.io/oss-fuzz-base/base-runner
Step #5: Running fuzz
Step #5: [2021-03-09 10:46:40,982 INFO] Finding shared libraries for targets (if any).
Step #5: [2021-03-09 10:46:40,994 INFO] Finished finding shared libraries for targets.
Step #5: [2021-03-09 10:46:41,111 INFO] Finding shared libraries for targets (if any).
Step #5: [2021-03-09 10:46:41,122 INFO] Finished finding shared libraries for targets.
Step #5: [0;31merror: /workspace/out/coverage/src/mozilla-central/gfx/qcms/fuzz_targets/fuzz_target_qcms.rs: No such file or directory
Step #5: [0m[0;31mwarning: The file '/src/mozilla-central/gfx/qcms/fuzz_targets/fuzz_target_qcms.rs' isn't covered.
Step #5: [0m[2021-03-09 10:46:41,258 DEBUG] Finished generating per-file code coverage summary.
Step #5: [2021-03-09 10:46:41,258 DEBUG] Generating file view html index file as: "/workspace/out/coverage/report/linux/file_view_index.html".
Step #5: Traceback (most recent call last):
Step #5:   File "/opt/code_coverage/coverage_utils.py", line 829, in <module>
Step #5:     sys.exit(Main())
Step #5:   File "/opt/code_coverage/coverage_utils.py", line 823, in Main
Step #5:     return _CmdPostProcess(args)
Step #5:   File "/opt/code_coverage/coverage_utils.py", line 780, in _CmdPostProcess
Step #5:     processor.PrepareHtmlReport()
Step #5:   File "/opt/code_coverage/coverage_utils.py", line 578, in PrepareHtmlReport
Step #5:     self.file_view_path)
Step #5:   File "/opt/code_coverage/coverage_utils.py", line 450, in GenerateFileViewHtmlIndexFile
Step #5:     self.GetCoverageHtmlReportPathForFile(file_path),
Step #5:   File "/opt/code_coverage/coverage_utils.py", line 423, in GetCoverageHtmlReportPathForFile
Step #5:     self._MapToLocal(file_path)), '"%s" is not a file.' % file_path
Step #5: AssertionError: "/src/mozilla-central/gfx/qcms/fuzz_targets/fuzz_target_qcms.rs" is not a file.
Step #5: ********************************************************************************
Step #5: Code coverage report generation failed.
Step #5: To reproduce, run:
Step #5: python infra/helper.py build_image qcms
Step #5: python infra/helper.py build_fuzzers --sanitizer coverage qcms
Step #5: python infra/helper.py coverage qcms
Step #5: ********************************************************************************
Finished Step #5
ERROR
```
</details>

It complains about not finding `/src/mozilla-central/gfx/qcms/fuzz_targets/fuzz_target_qcms.rs` which doesn't exist, however `/src/mozilla-central/gfx/qcms/fuzz/fuzz_targets/fuzz_target_qcms.rs` does exist.

I'm not sure how to fix this.

--- a/infra/base-images/base-builder/cargo
+++ b/infra/base-images/base-builder/cargo
@@ -31,13 +31,13 @@ fi
 
 if [ "$SANITIZER" = "coverage" ] && [ $1 = "fuzz" ]
 then
-    fuzz_src_abspath=`pwd`
-    export RUSTFLAGS="$RUSTFLAGS --remap-path-prefix fuzz_targets=$fuzz_src_abspath/fuzz_targets"
     # hack to turn cargo fuzz build into cargo build so as to get coverage
     # cargo fuzz adds "--target" "x86_64-unknown-linux-gnu"
     (
     # go into fuzz directory if not already the case
     cd fuzz || true
+    fuzz_src_abspath=`pwd`
+    export RUSTFLAGS="$RUSTFLAGS --remap-path-prefix fuzz_targets=$fuzz_src_abspath/fuzz_targets"
     # do not optimize with --release, leading to Malformed instrumentation profile data
     cargo build --bins
     # copies the build output in the expected target directory

